libraries/AP_DDS/Idl/geometry_msgs/msg/PoseStamped.idl

```diff
@@ -15,4 +15,4 @@ module geometry_msgs {
       geometry_msgs::msg::Pose pose;
     };
   };
-};
+};
```

Comment:
Missing newline at EOF.

Reply:
@peterbarker Are you ok if I enable the pre-commit hook to catch this on the IDL folder? (In a separate PR of course

Reply:
Fixed newline in cdeaf7b5c13d4b07b61daaeb27189bec767e004c